spec/models/evidence/forms/evidence_spec.rb

```diff
@@ -60,17 +60,52 @@
     end
   end
 
-  describe '#save' do
-    before do
-      allow(subject).to receive(:valid?).and_return(true)
-      allow(subject).to receive(:persist!)
-      allow(EvidenceCheck).to receive(:find)
-      allow_any_instance_of(EvidenceCheck).to receive(:update)
-      allow_any_instance_of(Reason).to receive(:save)
+  describe '#save', focus: true do
```

Comment:
By taking this approach, you've turned the unit test into an integration test and that can be done with the features rather than testing it here.

I'm not going to oppose this, as I'd rather see the feature complete. But I'd like to revisit this and try to implement it in a cleaner way without touching the database.